src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerArgumentCollection.java

```diff
@@ -50,10 +50,10 @@
      *
      * This argument allows you to set the GQ boundaries. HC expects a list of multiple GQ threshold values. To pass
      * multiple values, you provide them one by one with the argument, as in `-GQB 10 -GQB 20 -GQB 30` and so on. Note
-     * that GQ values are capped at 99 in the GATK.
+     * that GQ values are capped at 99 in the GATK, so values must be integers in the range [1, 99].
```

Comment:
Back at a computer---actually, it looks like I updated the documentation and changed the meaning of the GQB arguments in https://github.com/broadinstitute/gsa-unstable/pull/1451, along with changing the necessary tests.  These changes were not ported over in the GATK4 public/protected PRs by @ronlevine. so we should issue an additional PR to add these changes.